modules/network-observability-total-resource-usage.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+// * network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-total-resource-usage-table_{context}"]
+= Total average memory and CPU usage
+The following table outlines averages of total resource usage for clusters with a sampling value of 1, 50, and 400.
```

Comment:
```suggestion
The following table outlines averages of total resource usage for clusters with a sampling value of 1, 50, and 400 for 3 different tests: `Test 1`, `Test 2`, and `Test 3`. The tests differ in the following ways:
- Test 1 and 3 take into account the `namespaceCount`, `podStatusCount` and `serviceCount` metrics , and represents use cases with a high number of workloads.
- Test 2 takes into account a high ingress traffic throughput.

Since different types of cluster use cases are exemplified in the different tests, these numbers cannot be linearly compared side-by-side but instead used as a benchmark to evaluate what your personal cluster usage may be.
```

Reply:
Based on [Slack](https://redhat-internal.slack.com/archives/C02939DP5L5/p1720812777670899?thread_ts=1720793630.658669&cid=C02939DP5L5) thread, we need to add more context about whats in the tests as a kind of "expectation-setting" to introduce the table.

Reply:
cc @stleerh @memodi wdyt about adding some expectation-setting to the introduction, like this? Anything you would add here?

Reply:
@skrthomas thanks, overall it looks good to me. Instead of listing each prometheus metric names that we defined in our tests, I'd say `... take into account total number of Namespace, Pods and Services in OCP cluster ... with a high number of workloads for a given cluster size`

Reply:
Ok, cool. That makes sense, thank you!

Reply:
That looks good to me, thanks @skrthomas 
I think given the tests that we run, this is well formulated. But still, this is IMO not what users would expect when they look for reference values. I think for 1.7 we should run consistent tests for different cluster sizes (like NDH-cni) for this doc - even if we continue to run other tests internally for perf regression tracking.

Reply:
@jotak  so if I understand, this is as good as we can give with the tests that we run today? And there's future potential to run consistent tests for different cluster sizes?

Reply:
Yes exactly, I hope in 1.7 we can simplify the doc by showing data with the same test used for each cluster size

Reply:
Do you think we should wait to publish this until 1.7, and wait on those updates? Or should we add these now?